python/tests/catalog/test_base.py

```diff
@@ -78,14 +75,14 @@ def load_table(self, identifier: Union[str, Identifier]) -> Table:
         try:
             return self.__tables[identifier]
         except KeyError as error:
-            raise NoSuchTableError(f"Table does not exist: {identifier}") from error
+            raise NoSuchNamespaceError(f"Table does not exist: {identifier}") from error
```

Comment:
I don't think this change is correct.

Reply:
I went over them, and fixed them according to the spec